android/quest/build.gradle.kts

```diff
@@ -381,6 +381,8 @@ dependencies {
   implementation(libs.dagger.hilt.android)
   implementation(libs.hilt.work)
   implementation(libs.cql.measure.evaluator)
+  //  implementation("com.google.android.gms:play-services-location:{version}")
+  implementation("com.google.android.gms:play-services-location:21.0.1")
```

Comment:
Extract this to the version catalog. You can press `Alt + Enter` for a shortcut.